keep/api/core/db.py

```
@@ -3097,6 +3099,32 @@
     return query
 
 
+def enrich_incidents_with_alerts(tenant_id: str, incidents: List[Incident], session: Optional[Session]=None):
```

Ruff

`Incident` may be undefined, or defined from star imports

(F405)